downstream/modules/analytics/proc-add-roles-to-group.adoc

```diff
@@ -14,7 +14,7 @@ Adding roles to an existing group will grant new permissions to existing users i
 . Click image:images/cog.png[] > *Settings*.
 . From the sidebar, navigate to *User Access* > *Groups*.
 . Click the group you want to add users to.
```

Comment:
Minor grammar issue, suggest rewriting:

Click the group to which you want to add users.